src/duplo_resource/cronjob.py

```diff
@@ -8,38 +8,57 @@ class DuploCronJob(DuploTenantResourceV3):
   """
   Duplo CronJob are scheduled jobs that run containers in a Kubernetes cluster.
   """
-  
+
   def __init__(self, duplo: DuploClient):
     super().__init__(duplo, "k8s/cronJob")
 
   @Command()
-  def update_image(self, 
-                   name: args.NAME, 
+  def update_image(self,
+                   name: args.NAME,
                    image: args.IMAGE):
     """Update the image of a cronjob.
-    
+
     Args:
       name (str): The name of the cronjob to update.
       image (str): The new image to use for the cronjob.
     """
     cronjob = self.find(name)
+
+    # Set IsAnyHostAllowed
+    self._set_is_any_host_allowed(cronjob)
+
     cronjob["spec"]["jobTemplate"]["spec"]["template"]["spec"]["containers"][0]["image"] = image
     self.update(cronjob)
     return {"message": f"Successfully updated image for cronjob '{name}'"}
-  
+
   @Command()
-  def update_schedule(self, 
-                      name: args.NAME, 
+  def update_schedule(self,
+                      name: args.NAME,
                       cronschedule: args.CRONSCHEDULE) -> dict:
     """Update the schedule of a cronjob.
-    
```

Comment:
I'm thinking your editor did something funky here and makes this pr look way bigger than it is.